### PR TITLE
fix outdated eslint rule

### PR DESCRIPTION
**Pull request by olegakbarov**
Are you guys interested in more strict `.eslintrc`? I could work on it.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @jfurrow

**Comment by rcorral** [comment]
@olegakbarov thanks for your contribution! Would love to have you help us out with a more strict eslint file.

File: .eslintrc [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "space-before-function-paren": [2, {
       "anonymous": "always", "named": "never"
     }],
-    "spaced-line-comment": [2, "always"],
+    "spaced-comment": [2, "always"],
```
Comment (rcorral) [inline .eslintrc]: I'm not sure this will work since we're not using the 1.x version of eslint.

Comment (olegakbarov) [inline .eslintrc]: Yea, i got to check this. Are you're interested in migration towards >=1.0.0 ?

Comment (rcorral) [inline .eslintrc]: That sounds good, I think it makes sense!
I would just want to keep parity at the very minimum with the rules that we currently employ.

Comment (olegakbarov) [inline .eslintrc]: I took all of your rules and only added a few new. You can [take a look at the result](https://gist.github.com/olegakbarov/585bdf5232f181639487). What do you think?

**Comment by rcorral** [comment]
We've upgraded to Babel6

**Pull request closed without merging**